From a0037c7c4dbc8c7d642f8a686810addb7a3fcb1b Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 11 Jan 2018 11:00:53 +0200
Subject: [PATCH 06/25] legacy: allow rmt_storage sys_admin capability

Removed in ff2ffc9fa8f05e22d98ac2809dbd819a80d1cc39,
but still used in old blobs to set its own ioprio.

Change-Id: Ib67e787e91dc7648a5cea0be8cb619424a838a8b
---
 legacy-common/rmt_storage.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 legacy-common/rmt_storage.te

diff --git a/legacy-common/rmt_storage.te b/legacy-common/rmt_storage.te
new file mode 100644
index 0000000..7303b7e
--- /dev/null
+++ b/legacy-common/rmt_storage.te
@@ -0,0 +1 @@
+allow rmt_storage self:capability sys_admin;
-- 
2.7.4

